src/lib/Components/ArtworkGrids/__tests__/Artwork-tests.tsx

```diff
@@ -36,6 +36,11 @@ describe("in an open sale", () => {
     const artwork = renderer.create(<Artwork artwork={artworkProps(saleArtwork) as any} />).toJSON()
     expect(artwork).toMatchSnapshot()
   })
+
+  it("safely handles a missing sale_artwork", () => {
+    const artwork = renderer.create(<Artwork artwork={artworkProps(null) as any} />).toJSON()
+    expect(artwork).toMatchSnapshot()
+  })
```

Comment:
Maybe a better test is that running the renderer above doesn't crash?

Reply:
Yeah, good idea.